b9lab-content/4-running-a-chain/04-node-api-and-cli.md

```diff
@@ -1 +1,292 @@
+
+```
+
+Now, start your blockchain:
+
+```bash
+$ ./simd start
+
+6:23PM INF starting ABCI with Tendermint
+6:23PM INF Starting multiAppConn service impl=multiAppConn module=proxy
+6:23PM INF Starting localClient service connection=query impl=localClient module=abci-client
+6:23PM INF Starting localClient service connection=snapshot impl=localClient module=abci-client
+6:23PM INF Starting localClient service connection=mempool impl=localClient module=abci-client
+6:23PM INF Starting localClient service connection=consensus impl=localClient module=abci-client
+
+```
+
+You will see blocks producing and validating. Open a new terminal and check the balances:
+
+```bash
+$ ./simd query bank balances $(./simd keys show b9lab -a)
+balances:
+- amount: "30000000"
+  denom: stake
+pagination:
+  next_key: null
+  total: "0"
+```
+
+Create another key for a student and send it some tokens:
+
+```bash
+$ ./simd keys add student
+
+- name: student
+  type: local
+  address: cosmos1m95dh3uc2s7fkn4w6v3ueux3sya96dhdudwa24
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AgDYHucSs5vZ4viGyyoC0Qz6M7/+fEdqgOesEmeTdPE/"}'
+  mnemonic: ""
+
+
+**Important** record this mnemonic phrase in a safe place.
+It is the only way to recover your account if you ever forget your password.
+
+gown all scissors page panel table hill acoustic junior run winter cement mass clump moon adjust glare never satoshi easily illness hip rib multiply
+```
+
+Confirm the balance of the new account:
+
+```bash
+$ ./simd query bank balances $(./simd keys show student -a)
+balances: []
+pagination:
+  next_key: null
+  total: "0"
+```
+
+This account does not have a balance. In fact, it is not in the blockchain yet. We need to send a transaction to change this new accounts balance:
+
+```bash
+$./simd tx bank send $(./simd keys show b9lab -a) $(./simd keys show student -a) 10stake --chain-id test-chain-rT4wZY
+
+{"body":{"messages":[{"@type":"/cosmos.bank.v1beta1.MsgSend","from_address":"cosmos1nw793j9xvdzl2uc9ly8fas5tcfwfetercpdfqq","to_address":"cosmos1m95dh3uc2s7fkn4w6v3ueux3sya96dhdudwa24","amount":[{"denom":"stake","amount":"10"}]}],"memo":"","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":[]}
+
+confirm transaction before signing and broadcasting [y/N]: y           
+code: 0
+codespace: ""
+data: ""
+gas_used: "0"
+gas_wanted: "0"
+height: "0"
+info: ""
+logs: []
+raw_log: ""
+timestamp: ""
+tx: null
+txhash: D2CCFD91452F8C144BB1E7B54B9723EE3ED85925EE2C8AD843392721D072B895
+```
+
+As you can see, it will ask you to confirm the signing and broadcasting and shows you usefull information such as gas_used.
+
+Now check the balance of the student again:
```

Comment:
```suggestion
Now check the balance of the student account again:
```

Reply:
the student? hee hee 
(thinking of myself as a student, I had no balance!) just word choice here, let's be clear we're checking the account balance